corehq/apps/translations/app_translations.py

```diff
@@ -317,36 +327,36 @@ def get_bulk_app_sheet_rows(app, exclude_module=None, exclude_form=None):
         if exclude_module is not None and exclude_module(module):
             continue
 
-        module_string = "module" + str(mod_index + 1)
+        sheet_name = get_module_sheet_name(module)
         rows[MODULES_AND_FORMS_SHEET_NAME].append(_get_modules_and_forms_row(
             row_type="Module",
-            sheet_name=module_string,
+            sheet_name=sheet_name,
             languages=[module.name.get(lang) for lang in app.langs],
             media_image=[module.icon_by_language(lang) for lang in app.langs],
             media_audio=[module.audio_by_language(lang) for lang in app.langs],
             unique_id=module.unique_id,
         ))
 
-        rows[module_string] = []
+        rows[sheet_name] = []
         if not isinstance(module, ReportModule):
-            rows[module_string] += _get_module_case_list_form_rows(app.langs, module)
-            rows[module_string] += _get_module_detail_rows(app.langs, module)
+            rows[sheet_name] += _get_module_case_list_form_rows(app.langs, module)
+            rows[sheet_name] += _get_module_detail_rows(app.langs, module)
 
             for form_index, form in enumerate(module.get_forms()):
                 if exclude_form is not None and exclude_form(form):
                     continue
 
-                form_string = module_string + "_form" + str(form_index + 1)
+                sheet_name = get_form_sheet_name(form)
```

Comment:
Reusing this variable name (`sheet_name`) is confusing. What do you think of renaming it?

Reply:
f1ac43d5a1b988a5da951f48edc05481c329dc6c